src/components/Lookup/index.js

```diff
@@ -236,6 +243,63 @@ class Lookup extends Component {
         }
     }
 
+    // -- mine
+
+    stopArrowScoll() {
+        if (this.scrollingTimer) {
+            clearInterval(this.scrollingTimer);
+        }
+    }
+
+    scrollTo(offset) {
+        const menu = this.menuRef.current.getRef();
+        menu.scrollTo(0, offset);
+    }
+
+    scrollBy(offset) {
+        const menu = this.menuRef.current.getRef();
+        menu.scrollBy(0, offset);
+    }
+
+    handleScrollUpArrowHover() {
+        this.stopArrowScoll();
+
+        const menu = this.menuRef.current.getRef();
+        this.scrollingTimer = setInterval(() => {
+            if (menu.scrollTop > 0) {
+                menu.scrollBy(0, -1);
+            } else {
+                this.stopArrowScoll();
+            }
+        }, 5);
+    }
+
+    handleScrollDownArrowHover() {
+        this.stopArrowScoll();
+
+        const menu = this.menuRef.current.getRef();
+        this.scrollingTimer = setInterval(() => {
+            if (!isScrollPositionAtMenuBottom(menu)) {
+                menu.scrollBy(0, 1);
+            } else {
+                this.stopArrowScoll();
+            }
+        }, 5);
+        this.updateScrollingArrows();
+    }
+
+    updateScrollingArrows() {
+        const menu = this.menuRef.current.getRef();
+        const showScrollUpArrow = menu.scrollTop > 0;
+        const showScrollDownArrow = !isScrollPositionAtMenuBottom(menu);
+        this.setState({
+            showScrollUpArrow,
+            showScrollDownArrow,
+        });
+    }
+
+    // - end of mine
```

Comment:
since we are using the same logic in other components or it can potentially be used then we can move it out of the component into a library, this behavior doesn't have to be tied to react, it's pure DOM manipulation of the scrollable container.